src/portable/portable.c

```diff
@@ -1126,7 +1128,7 @@ static int attach_unit_file(
                 _cleanup_(unlink_and_freep) char *tmp = NULL;
                 _cleanup_close_ int fd = -1;
 
-                fd = open_tmpfile_linkable(path, O_WRONLY|O_CLOEXEC, &tmp);
+                fd = open_tmpfile_linkable_label(path, O_WRONLY|O_CLOEXEC, &tmp);
```

Comment:
We've gone in a different direcion, so please drop this change - the rest is good